### PR TITLE
Added support for jboss-cli

### PR DESCRIPTION
**Pull request by rparree**

### CODE REVIEW

**Comment by rparree** [comment]
will add relevance: 0

**Comment by rparree** [comment]
I am getting a unresponsive page when i try to use within the browser.

Works fine when i access `build/demo/index.html` when building with only `node tools/build.js -n jboss-cli`. When building with `node tools/build.js` my browser hangs.

**Comment by rparree** [comment]
In fact it works fine in the browser as long as i use the `--no-compress` option. After compressing the page becomes unresponsive

File: src/languages/jboss-cli.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ Category: config
+ */
+
+function (hljs) {
+  var PARAMS = [{
+    className: "attribute",
+    begin: /[\w-]+\s?=/,
+    starts: {
+
+      ends: /$/,
```
Comment (isagalaev) [inline src/languages/jboss-cli.js]: Should be `end` I guess?

File: src/languages/jboss-cli.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      className: "string"
+    },
+    relevance: 1
```
Comment (isagalaev) [inline src/languages/jboss-cli.js]: 1 is the default value, so shouldn't be set explicitly (JavaScript being a weird language where we care about code size). Same for all the other occurrences of it down the file.

File: src/languages/jboss-cli.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  var COMMAND_PARAMS = {
+    className: 'params',
+    begin: "--",
+    end: "$",
+    relevance: 1
+
+  };
+  return {
+    aliases: ['wildfly-cli'],
+    case_insensitive: false,
```
Comment (isagalaev) [inline src/languages/jboss-cli.js]: This one is also unnecessary, since `false` is the default.

**Comment by rparree** [comment]
fixed the problems, but now i have a problem that the `PARAMSBLOCK` (`begin: /\(/,end: /\)/`) which contains `PARAMS` is very eager and consumes all content until the last closing parenthesis. I will look into this today (working off another branch)

**Comment by isagalaev** [comment]
Merged with a few addition tweaks. Thanks!

**Pull request merged**